### ISSUE TITLE
The items do not follow the hands if an animation is played

### ISSUE THREAD

**Issue by EchoShax** [Yesssssman/epicfightmod#1236]
### Have you checked if a similar issue is already reported by someone else?

- [X] I checked there are no similar issues have been reported.

### Have you read the support policy?

- [X] I read it and I accept the policy.

### Are you using the latest Epic Fight and recommended Forge version?

- [X] I checked I'm using latest Epic Fight and recommended Forge version.

### Is this issue related to mod incompatibility?

- [ ] This is a mod compatibility issue and I'm aware of the problem.

### Minecraft Version

1.16.5

### What happened?

Basically, if i hit without any items in my hand, nothing is wrong. But, i if i hold an item and hit (ex: trying to hit a mob with a diamond sword), the item stays in the same place, and the item doesnt move, but the animation still plays. What do i do to fix this? I am doing it at EpicFight 16.6.1 and recommended forge ver, since newest version literally breaks even more. Please help me :)

**Comment by drwad** [comment]
Same happens to me but in 1.18.2 version :/  specifically in EpicFight-18.3.7.jar ver.

**Comment by MetalKnight56** [comment]
> ### Have you checked if a similar issue is already reported by someone else?
> * [x]  I checked there are no similar issues have been reported.
> 
> ### Have you read the support policy?
> * [x]  I read it and I accept the policy.
> 
> ### Are you using the latest Epic Fight and recommended Forge version?
> * [x]  I checked I'm using latest Epic Fight and recommended Forge version.
> 
> ### Is this issue related to mod incompatibility?
> * [ ]  This is a mod compatibility issue and I'm aware of the problem.
> 
> ### The mod
> _No response_
> 
> ### Minecraft Version
> 1.16.5
> 
> ### What happened?
> Basically, if i hit without any items in my hand, nothing is wrong. But, i if i hold an item and hit (ex: trying to hit a mob with a diamond sword), the item stays in the same place, and the item doesnt move, but the animation still plays. What do i do to fix this? I am doing it at EpicFight 16.6.1 and recommended forge ver, since newest version literally breaks even more. Please help me :)

Update to newer versions
